media-libs/oidn/oidn-1.4.1.ebuild

```
@@ -0,0 +1,39 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{6,7,8,9} )
```

Lot of obsolete implementations here (which the eclass ignores anyway):
```suggestion
PYTHON_COMPAT=( python_{8,9} )
```